src/caret_analyze/plot/bokeh/pub_sub_info_interface.py

```diff
@@ -55,21 +55,27 @@ def show(
 
         # TODO: self._pub_subs is not defined.
         all_topic_names = sorted({ps.topic_name for ps in self._pub_subs})
-        # not interactive
-        if self._last_export_path:
-            self._show_core('All')
-            return None
-        if len(all_topic_names) == 1:
+        if self._last_export_path or len(all_topic_names) == 1:
             return [self._show_core('All')]
-        elif not interactive:
-            return [self._show_core(topic_name) for
-                    topic_name in ['All'] + all_topic_names]
-        # interactive
         else:
-            topic_dropdown = Dropdown(description='Topic:',
-                                      options=(['All'] + all_topic_names))
-            interact(self._show_core,
-                     topic_name=topic_dropdown)
+            [self._show_core(topic_name) for
+             topic_name in ['All']+all_topic_names]
```

Comment:
Only PubSubTimeSeriesPlot visualize multiple figures at once.
Couldyou you please align this with the other Plots?

Also, return is missing.

```suggestion
            return [self._show_core('All')]
```